requirements/test.in

```diff
@@ -12,7 +12,7 @@ pytest-cov
 pytest-mock
 pytest-xdist
 pytest_codspeed
-python-on-whales
+python-on-whales; python_version < "3.14" # no wheel for 3.14
```

Comment:
This doesn't seem reasonable.

Reply:
python-on-whales is a already an optional dependency as it is imported by `pytest.importorskip`
https://github.com/aio-libs/aiohttp/blob/911645731da52f0572484b86729b2dab141a090f/tests/autobahn/test_autobahn.py#L13

Would you like to handle this differently?

Reply:
Well, since it's here, it actually gets installed into all the test envs, making it not really optional de-facto. So maybe it should be treated as such in the tests too. Anyway, something like this is probably better in a dedicated PR.

Reply:
Okay, I am leaving this one as it is then.

Reply:
As mentioned elsewhere, pydantic should have a release in a few days, so I think we should wait this one out and then revert these changes. We'll also see if there's any further update from isal in that time too, hopefully rendering this PR unnecessary.

Reply:
It's been almost two weeks and there hasn't been any release of pydantic or isal. @Dreamsorcerer  do you want to make these optional? if not I can close this PR for now and come back when all the projects are released.

Reply:
pydantic is not a direct dependency so changing the version isn't easy, it would likely conflict with python-on-whales

Reply:
The version is just pinned by pip-tools, so we can just change that lock and then Dependabot will update it to the final release when it's available.